redata/backends/exasol.py

```diff
@@ -0,0 +1,259 @@
+import datetime
+import decimal
+from types import SimpleNamespace
+from urllib.parse import urlparse
+
+import pyexasol
+from redata.backends.base import DB
+
+
+class ExasolEngine(object):
+    def __init__(self, url):
+        self.creds = parse_url(url)
+
+    def execute(self, *args, fetch_dict=False, **kwargs):
```

Comment:
possibly for would make fetch_dict=True by default, as it seems to be used like that in most cases